src/v2/Components/NavBar/Menus/DropDownMenu/DropDownMenu.tsx

```diff
@@ -75,38 +48,67 @@ export const DropDownNavMenu: React.FC<DropDownNavMenuProps> = ({
           "center",
         ]}
       >
-        <Links py={3} mr={[3, 3, "50px", "50px"]}>
-          <Box
-            display="flex"
-            flexDirection="column"
-            height="100%"
-            mr={[1, 1, 2, 2]}
-            width={[110, 135, 135, 170, 170]}
-          >
-            {menu.links.map((menuItem, i) => {
-              if (!("menu" in menuItem)) {
-                const isLast = menu.links.length - 1 === i
+        <Flex width={breakpoints.xl} px={3}>
```

Comment:
This is the only real change, which caps the width of the nav bar to the max width of our page content